ai_powered_qa/components/agent.py

```diff
@@ -178,10 +177,36 @@ def _get_messages_for_completion(
         if user_prompt:
             messages.append({"role": "user", "content": user_prompt})
 
-        print(messages[1])
-
         return messages
 
     def _generate_context_message(self):
         contexts = [p.context_message for p in self.plugins.values()]
         return "\n\n".join(contexts)
+
+    def generate_whisperer_interaction(
```

Comment:
Just so we can merge this into main, I would move this outside of the Agent class. It's highly specific to the Gherkin agent, so I think we could just have this hardcoded in the UI script. Either just as a simple OpenAI call, or initializing two agents (main one with the PlaywrightPlugin, and a second one with this system message, where we don't need to save the history).